stable/insights-agent/requirements.lock

```diff
@@ -1,15 +0,0 @@
-dependencies:
```

Comment:
mmh why was this file deleted?

Reply:
We unpinned the versions so that we don't have to keep updating this chart every time a downstream has a patch.